plugins/woocommerce/src/Blocks/Domain/Bootstrap.php

```diff
@@ -387,28 +388,28 @@ function () {
 		$this->container->register(
 			'Automattic\WooCommerce\Blocks\StoreApi\Formatters',
 			function( Container $container ) {
-				$this->deprecated_dependency( 'Automattic\WooCommerce\Blocks\StoreApi\Formatters', '7.2.0', 'Automattic\WooCommerce\StoreApi\Formatters', '7.4.0' );
+				$this->deprecated_dependency( 'Automattic\WooCommerce\Blocks\StoreApi\Formatters', '6.4.0', 'Automattic\WooCommerce\StoreApi\Formatters', '6.5.0' );
```

Comment:
Those should be removed honestly, maybe in a follow up.

Reply:
let's make sure there is a ticket for this removal if it's necessary

Reply:
Is it safe to remove those? Or should it be considered a breaking change?

Reply:
They're deprecated for a solid 2 years now, I think it's safe to delete. I will open a follow up.